### ISSUE TITLE
Appgrid the right way

### ISSUE THREAD

**Issue by chibenwa** [linagora/tmail-flutter#3260]
# Summary

Because the app grid is currently hard coded we are better off removing it: https://github.com/linagora/tmail-flutter/issues/3258

Yet it is usefull to see 'related apps' to ease their instalation.

Backend offered a means for admins to hint users into chossing related apps using `.well-known/linagora-ecosystem`: https://github.com/linagora/tmail-backend/issues/1289

When disabled show no mobile app grid.

When enabled show the mobile app grid based on it:
 - when clicked open the app if installed
 - and if not installed I am taken to the app store

**Comment by chibenwa** [comment]
Cf @hoangdat @guimard

**Comment by hoangdat** [comment]
@chibenwa 
To be confirmed: `.well-known/linagora-ecosystem` only use for app grid mobile? Am I right?

**Comment by chibenwa** [comment]
>  To be confirmed: .well-known/linagora-ecosystem only use for app grid mobile? Am I right?

Yes. For web we can keep relying on static assets configuration in the Single Page Application I believe.

**Comment by hoangdat** [comment]
- [ ] if an app existed in `mobile-app`, priority to show icon with link to mobile app 
  - iOS vs Android
- [ ] if an app not existed in `mobile-app`, show icon with link to web app (avoid duplication)

**Comment by hoangdat** [comment]
![Image](https://github.com/user-attachments/assets/c94a9064-6ebd-448d-98f5-6dd0cd78332a)

**Comment by hoangdat** [comment]
```
{
  "Twake Drive": {
  },
  "Twake Chat": {
  },
  "Twake Sync" {
  }
}
```

**Comment by dab246** [comment]
> ```
> {
>   "Twake Drive": {
>   },
>   "Twake Chat": {
>   },
>   "Twake Sync" {
>   }
> }
> ```

IMO, We don't need to change like that. Keeping the format of the returned BE intact, we can still parse it.
